CMakeLists.txt

```diff
@@ -117,6 +117,10 @@ install(
 if(WINDOWS)
     # set PATH to DLLs on Windows
     set(DLL_PATH_LIST "PATH=path_list_append:../bin/$<CONFIG>")
+    # add path to the proxy lib DLL
+    set(DLL_PATH_LIST "${DLL_PATH_LIST};PATH=path_list_append:../src/proxy_lib")
```

Comment:
nit: Could we use `${CMAKE_CURRENT_SOURCE_DIR}` or other predefined CMake `DIR` variables to refer to those directories?

Reply:
How do you want to do that? Did you try it? Does it work for you?

Reply:
I didn't test that, it's just that we usually refer to CMake vars when setting appropriate paths.

Reply:
I believe the `../` part is disturbing a little - we don't know exactly how this behaves if cmake command would be run from some different place... but we can leave that as is, for now, I guess

In general we should probably use `${PROJECT_BINARY_DIR}`, but it would require checking the structure of dirs in Windows builds

Reply:
@kswiecicki  @lukaszstolarczuk  Done

Reply:
Thx!